06_puppy/kasunfdo/types.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"sync"
+)
+
+// Data structure for puppy
+type Puppy struct {
+	ID     uint64
+	Breed  string
+	Colour string
+	Value  float64
+}
+
+// Storer defines standard CRUD operations for Puppy
```

Comment:
This is _not quite_ the right format for a doc string. It needs a full stop at the end to make it a complete sentence.